ts/features/pn/aar/saga/fetchQrCodeSaga.ts

```diff
@@ -43,10 +44,20 @@ export function* fetchAARQrCodeSaga(
                 fullNameDestinatario: recipientInfo.denomination
               };
               return setAarFlowState(nextState);
+            case 403:
+              const notAddresseeFinalState: AARFlowState = {
+                type: sendAARFlowStates.notAddresseeFinal,
+                iun: data.value.iun,
+                fullNameDestinatario: data.value.recipientInfo.denomination,
+                qrCode: qrcode
+              };
+              return setAarFlowState(notAddresseeFinalState);
+
             default:
               const errorState: AARFlowState = {
                 type: sendAARFlowStates.ko,
-                previousState: currentState
+                previousState: currentState,
+                errorCodes: data.value.errors?.map(x => x.code) ?? []
```

Comment:
See comment below (above?) on including the entire `errors` structure. Also, the `traceId`